### PR TITLE
Include locations for release notifications in the readme.

### LINKED ISSUE #4266: Document how to find out about new releases
### Description
Add a section to the README and possibly user docs on where to get notifications of new Teku releases from.  Twitter, email, discord I think are the main options.

### PR DESCRIPTION
**Pull request by rolfyone**
fixes #4266

Signed-off-by: Paul Harris <paul.harris@consensys.net>

## Documentation

- [X] I thought about documentation and added the `documentation` label to this PR if updates are required.

## Changelog

- [X] I thought about adding a changelog entry, and added one if I deemed necessary.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -38,6 +38,12 @@ or [zip format](https://artifacts.consensys.net/public/teku/raw/names/teku.zip/v
 We recommend only using release versions for MainNet, but `develop` builds are useful for testing 
 the latest changes on testnets.
 
+Release notifications are available via:
+* `teku` in [Consensys Discord](https://discord.gg/7hPv2T6), 
+* `client-updates` in [Ethstaker Discord](https://discord.gg/u2q7FRv)
```
Comment (ajsutton) [inline README.md]: I probably wouldn't list this one as an official source. It is useful but we don't actually control it so can't do anything if they stop including our announcements there in the future.

File: README.md [hunk 2]
```diff
@@ -38,6 +38,12 @@ or [zip format](https://artifacts.consensys.net/public/teku/raw/names/teku.zip/v
...
+* Subscribe to release notifications on github for [teku](https://github.com/ConsenSys/teku)
+* Follow us on [Twitter](https://twitter.com/Teku_ConsenSys)
+* Sign up to our [release announcements](https://pages.consensys.net/teku-sign-up) email list (release and important announcements only, no marketing)
```
Comment (ajsutton) [inline README.md]: Probably worth listing the email list first, then twitter, then our discord and finally GitHub release announcements. Probably arbitrary but feels like it puts the more reliable but less discoverable channels first.

**Review by ajsutton — APPROVED** [review]
LGTM.

File: util/src/main/java/tech/pegasys/teku/util/config/Constants.java [hunk 3]
```diff
@@ -116,7 +116,7 @@
 
   // Teku Networking Specific
   public static final int VALID_BLOCK_SET_SIZE = 1000;
-  public static final int VALID_ATTESTATION_SET_SIZE = 1000;
+  public static final int VALID_ATTESTATION_SET_SIZE = 21000;
```
Comment (ajsutton) [inline util/src/main/java/tech/pegasys/teku/util/config/Constants.java]: I take that back, this appears to have been pushed to the wrong PR. :)

**Review by ajsutton — CHANGES_REQUESTED** [review]
See stray change comment.

File: util/src/main/java/tech/pegasys/teku/util/config/Constants.java (hunk 3 above)
Comment (rolfyone) [reply util/src/main/java/tech/pegasys/teku/util/config/Constants.java]: definitely the wrong PR...

**Review by ajsutton — APPROVED** [review]
Now it looks good. :)

**Pull request merged**